app/views/pages/conditions/delete.html.erb

```diff
@@ -0,0 +1,46 @@
+<% set_page_title(title_with_error_prefix(t('page_titles.routing_page'), condition_form.errors&.any?)) %>
+<% content_for :back_link, govuk_back_link_to(edit_condition_path(condition_form.form.id, condition_form.page.id, condition_form.record.id)) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= form_with(model: condition_form, url: delete_condition_path(condition_form.form.id, condition_form.page.id, condition_form.record.id), method: :delete) do |f| %>
+      <% if condition_form&.errors.any? %>
```

Comment:
On searching forms-admin it looks like we use both in various places. I'm not sure we actually need either - I think we always pass a `Pages::ConditionsForm` to this view, and since that model includes `ActiveModel::Validations` it should always have an `errors` method.

Reply:
I've removed it, can't seem to find a reason we've put it there!